docs/guides/getting-started.md

```diff
@@ -90,18 +93,21 @@ With spot/preemptible instances (`spot >= 0`), auto-recovery logic and persisten
 ## Query Status
 
 ```console
-$ terraform refresh && terraform show
+$ TF_LOG_PROVIDER=INFO terraform refresh
+$ TF_LOG_PROVIDER=INFO terraform show
 ```
 
 These commands will:
 
 1. Query the task status from the cloud.
 2. Display the task status.
 
+-> **Note**: Use `TF_LOG_PROVIDER=DEBUG` in lieu of `INFO` to increase verbosity for debugging. See the [logging docs](https://www.terraform.io/plugin/log/managing) for a full list of options.
```

Comment:
is https://www.terraform.io/internals/debugging a better link? idk

Reply:
🤷🏼‍♂️ https://www.terraform.io/plugin/log/managing is more comprehensive; thoroughly documented capital sins have more chances of being forgiven.

Reply:
Note: this note might not be needed after #471

Reply:
whoa #471 looks [excessive](https://github.com/iterative/terraform-provider-iterative/pull/471#discussion_r840077140)

Reply:
Note: by relevant I mean _relevant_, not _all_.